rel/server/env.sh.eex

```diff
@@ -12,6 +12,18 @@ if [ "$LIVEBOOK_CLUSTER" = "auto" ] && [ ! -z "$FLY_APP_NAME" ]; then
     deployment="$(echo "$FLY_IMAGE_REF" | shasum | cut -c 1-10)"
     export LIVEBOOK_NODE="${FLY_APP_NAME}-${deployment}@${FLY_PRIVATE_IP}"
   fi
+elsif [ ! -z "$LIVEBOOK_CLUSTER" ] && [ ! -z "$AWS_EXECUTION_ENV" ] ; then
+  # This covers any AWS service platform: EC2, ECS, and Fargate
```

Comment:
As the env var indicates that it covers ECS. EC2 and Fargate are just capacity providers for the cluster :)

```suggestion
```